packages/blade/src/components/Link/_decisions/decisions.md

```diff
@@ -0,0 +1,33 @@
+# Link Decisions <!-- omit in toc -->
+- [Design](#design)
+- [Link Component](#link-component)
+  - [Link API](#link-api)
+- [Component Breakdown](#component-breakdown)
+- [Open Questions](#open-questions)
+
+## Design
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=12699%3A147155) to the **Link** component
+
+
+## Link Component
+A Link component that can act as a `<button>` as well as an `<anchor>` element
+
+### Link API
```

Comment:
How would we define external links with `target=__blank` we might need another prop like `isExternal` which will add that in.

Reply:
Yup. This needs to be added. Let me think this through

Reply:
Should also add noopener noreferrer I suppose with external link.

Reply:
Would it also accept `rel` prop for noopener noreferrer?

Reply:
Added!